keyboards/hator/gravity/tkl/keyboard.json

```diff
@@ -0,0 +1,289 @@
+                { "label": "C", "matrix": [4, 4], "x": 4.25, "y": 4.25 },
+                { "label": "V", "matrix": [4, 5], "x": 5.25, "y": 4.25 },
+                { "label": "B", "matrix": [4, 6], "x": 6.25, "y": 4.25 },
+                { "label": "N", "matrix": [4, 7], "x": 7.25, "y": 4.25 },
+                { "label": "M", "matrix": [4, 8], "x": 8.25, "y": 4.25 },
+                { "label": "<", "matrix": [4, 9], "x": 9.25, "y": 4.25 },
+                { "label": ">", "matrix": [4, 10], "x": 10.25, "y": 4.25 },
+                { "label": "?", "matrix": [4, 11], "x": 11.25, "y": 4.25 },
+                { "label": "Shift", "matrix": [4, 13], "x": 12.25, "y": 4.25, "w": 2.75 },
+                { "label": "Up", "matrix": [4, 15], "x": 16.25, "y": 4.25 },
+
+                { "label": "Ctrl", "matrix": [5, 0], "x": 0, "y": 5.25, "w": 1.25 },
+                { "label": "Win", "matrix": [5, 1], "x": 1.25, "y": 5.25, "w": 1.25 },
+                { "label": "Alt", "matrix": [5, 2], "x": 2.5, "y": 5.25, "w": 1.25 },
+                { "label": "Space", "matrix": [5, 6], "x": 3.75, "y": 5.25, "w": 6.25 },
+                { "label": "Alt", "matrix": [5, 10], "x": 10, "y": 5.25, "w": 1.25 },
+                { "label": "Win", "matrix": [5, 11], "x": 11.25, "y": 5.25, "w": 1.25 },
+                { "label": "Fn", "matrix": [5, 12], "x": 12.5, "y": 5.25, "w": 1.25 },
+                { "label": "Ctrl", "matrix": [5, 13], "x": 13.75, "y": 5.25, "w": 1.25 },
+                { "label": "Left", "matrix": [5, 14], "x": 15.25, "y": 5.25 },
+                { "label": "Down", "matrix": [5, 15], "x": 16.25, "y": 5.25 },
+                { "label": "Right", "matrix": [5, 16], "x": 17.25, "y": 5.25 },
+                { "label": "NO_PIN", "matrix": [ 6, 0 ], "x": 0, "y": 0 },
+                { "label": "NO_PIN", "matrix": [ 6, 1 ], "x": 0, "y": 1 },
+                { "label": "NO_PIN", "matrix": [ 6, 2 ], "x": 0, "y": 2 },
+                { "label": "NO_PIN", "matrix": [ 6, 3 ], "x": 0, "y": 3 },
+                { "label": "NO_PIN", "matrix": [ 6, 4 ], "x": 0, "y": 4 },
+                { "label": "NO_PIN", "matrix": [ 6, 5 ], "x": 0, "y": 5 },
+                { "label": "NO_PIN", "matrix": [ 6, 6 ], "x": 0, "y": 6 },
+                { "label": "NO_PIN", "matrix": [ 6, 7 ], "x": 0, "y": 7 },
+                { "label": "NO_PIN", "matrix": [ 6, 8 ], "x": 17.5, "y": 0 },
+                { "label": "NO_PIN", "matrix": [ 6, 9 ], "x": 17.5, "y": 1 },
+                { "label": "NO_PIN", "matrix": [ 6, 10 ], "x": 17.5, "y": 2 },
+                { "label": "NO_PIN", "matrix": [ 6, 11 ], "x": 17.5, "y": 3 },
+                { "label": "NO_PIN", "matrix": [ 6, 12 ], "x": 17.5, "y": 4 },
+                { "label": "NO_PIN", "matrix": [ 6, 13 ], "x": 17.5, "y": 5 },
+                { "label": "NO_PIN", "matrix": [ 6, 14 ], "x": 17.5, "y": 6 },
+                { "label": "NO_PIN", "matrix": [ 6, 15 ], "x": 17.5, "y": 7 }
```

Comment:
These look like underglow LEDs at the left and right side of the keyboard. Why are they declared as also being part of the key matrix?